easybuild/easyblocks/t/tensorflow.py

```diff
@@ -218,7 +218,8 @@ def extra_options():
             # see https://developer.nvidia.com/cuda-gpus
             'cuda_compute_capabilities': [[], "List of CUDA compute capabilities to build with", CUSTOM],
             'path_filter': [[], "List of patterns to be filtered out in paths in $CPATH and $LIBRARY_PATH", CUSTOM],
-            'with_jemalloc': [None, "Make TensorFlow use jemalloc (usually enabled by default)", CUSTOM],
+            'with_jemalloc': [None, "Make TensorFlow use jemalloc (usually enabled by default)." +
```

Comment:
Missing space at the end:

```suggestion
            'with_jemalloc': [None, "Make TensorFlow use jemalloc (usually enabled by default). " +